### ISSUE TITLE
Email Tracking

### ISSUE THREAD

**Issue by s114sharma** [Pardot/api-docs#126]
Hi All,

Is their any option like as follows regarding Pardot.
1. In my org I had enable to W2L and then it will create the lead. When ever lead is inserted into my org then we fire the workflow email alert to that lead emails .
Now here , Is any possibility if i put some tracking code (pardot code) and add this code into the email template so when ever lead user open that email and open any web link so i can track this lead user on pardot.

In short we need to send email from salesforce and track that into the pardot.

Thanks in Advanced.
